app/presenters/document_presenter.rb

```diff
@@ -79,6 +79,10 @@ def footer_date_metadata
     end
   end
 
+  def bulk_published
+    document.details.metadata["bulk_published"]
```

Comment:
Is `document.details.metdata` a hash here or can we call it as `.bulk_published`?